lib/sqs.js

```diff
@@ -99,16 +125,33 @@ function queueAlreadyExists(queueName, attributes) {
   }
 }
 
+/**
+ * Clears all queues.
+ */
+
 function clearQueues() {
   queues.clear();
 }
 
+/**
+ * Gets state of queue.
+ *
+ * @param {string} QueueUrl - The url for the queue.
+ */
+
 function getQueueState(QueueUrl) {
   const convertedQueue = QueueUrl.replace(QUEUE_URL_REGEX, '');
 
   return queues.get(convertedQueue);
 }
 
+/**
+ * Creates a queue.
+ *
+ * @param {Object} params - Parameters for the queue.
```

Comment:
When we have a complex object like this we want to create a typedef and document every parameter inside
@param {CreateQueueParams}  params
.
.
@typedef {Object} CreateQueueParams
@property {string} queueName ...